main.js

```diff
@@ -36,7 +37,7 @@ app.on('ready', function () {
   }
   // Show dev tools
   // Remove this line before distributing
-  // win.webContents.openDevTools()
+  win.webContents.openDevTools()
```

Comment:
Is there a way to wrap this line in something like `if (running production) then`?